### PR TITLE
obs_basic_workflow: explain how to "move" a file

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/openSUSE/obs-docu/issues/236

### CODE REVIEW

File: xml/obs_basic_workflow.xml [hunk 1]
```diff
@@ -154,14 +154,15 @@
  <sect1 xml:id="sec.obs.basicworkflow.create-new">
   <title>Creating a New Package</title>
   <para>
-   This section covers how to create packages from a project hosted on &gh;
-   (the <quote>upstream project</quote>). We assume that this project contains source code
-   which you want to package for different &suse; distributions.
+   This section covers how to create packages from an arbitrary software project,
```
Comment (hennevogel) [inline xml/obs_basic_workflow.xml:157]:
```suggestion
   This section covers how to create packages for an arbitrary software project,
```

File: xml/obs_basic_workflow.xml [hunk 2]
```diff
@@ -312,6 +313,35 @@ Fri Aug 23 08:42:42 UTC 2017 - &exampleuser_mail;</screen>
     <screen>&prompt.user;<command>osc</command> commit</screen>
    </step>
   </procedure>
+  <para>
+   To delete a file in your working directory, merely deleting it from the local
+   filesystem ("rm FILENAME") is probably not going to be sufficient, since
```
Comment (hennevogel) [inline xml/obs_basic_workflow.xml:318]: Probably? You know for sure that is will not be sufficient once it's version controlled :)

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: xml/obs_basic_workflow.xml (hunk 2 above)
Comment (smithfarm) [reply xml/obs_basic_workflow.xml:318]: You're right: version control has no concept of sarcasm ;-)

**Comment by smithfarm** [comment]
@hennevogel Thanks for the review. Please have another look?

**Review by hennevogel — APPROVED** [review]

**Pull request merged**